src/mlpack/methods/ann/layer/add_merge_impl.hpp

```diff
@@ -23,6 +23,110 @@
 namespace mlpack {
 namespace ann /** Artificial Neural Network. */ {
 
+template<typename InputDataType, typename OutputDataType,
+typename... CustomLayers>
+AddMerge<InputDataType, OutputDataType, CustomLayers...>::
+AddMerge(const AddMerge& layer) :
+model(layer.model),
+run(layer.run)
+{
+  // Nothing to do here
+}
+
+template<typename InputDataType, typename OutputDataType,
+typename... CustomLayers>
+AddMerge<InputDataType, OutputDataType, CustomLayers...>::
+AddMerge(const AddMerge&& layer) :
```

Comment:
I think this argument doesn't need to be `const`. :+1:

Reply:
Okay.